nodegroups.tf

```diff
@@ -94,25 +93,9 @@ resource "aws_launch_template" "eks_node_groups" {
   }
 
   monitoring {
-    enabled = true
+    enabled = var.monitoring
```

Comment:
We can use a more descriptive variable name here, for example: `var.enable_detailed_monitoring`

Reply:
Done